active/0000-plaforms.md

```diff
@@ -0,0 +1,328 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+The ember-cli platform flag treats platforms as build targets, and gives you a
+workflow for producing builds customized to a platform, as well as streamlined tooling,
+enhanced resuability, and better separation of concerns for applications that
+target more than one platform.  Platforms are roughly equivalent to web views,
+or a shell which provides a web view, such as a browser.
+
+# Motivation
+
+There is a growing trend of shipping Ember applications inside of WebView shells to
+provide installable cross-platform applications and to gain access to native features.
+
+Currently, each platform has a homegrown solution for managing build/serve/test/deploy
+processes.  We could reduce the surface area of commands these platforms expose and
+ease the building of addons specific to a platform by enabling ember-cli to understand
+the existence of platforms and how various assets relate to them.
+
+# Detailed design
+
+## Usage
+
+These are the docs for using `ember-cli-platforms` with an installed platform plugin.
+For the docs on creating a platform plugin, [skip ahead](./README.md#creating-plugins).
+
+**Create a platform.**
+```cli
+ember platform <type> [-t "<name>"]
+```
```

Comment:
A good point.  A key distinction is that a platform does far more than generate `test` and `app` files, you also get a platform addon similar to an `in-repo-addon` and a platform directory (e.g. `platforms/cordova-ios`.

Another potential distinction is that it may be beneficial to have `platform` have `install` behavior as well.  If a given platform addon is not already present (say `ember-cli-platform-cordova`), then ember platform would fetch and install it.  Whether this is valuable, I'm less certain, and I'd be comfortable falling back to `generate` over increasing surface area.